stdlib/all_jlls.version

```diff
@@ -0,0 +1,29 @@
+# -*- mode: make -*-
```

Comment:
Ooops, it should have been 
```suggestion
# -*- mode: makefile -*-
```
like https://github.com/JuliaLang/julia/blob/7a1e4046d5ef005a8dd418b2071a1688cf00a34c/Make.inc#L1